postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -11,6 +12,11 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
   import this.FunctionDef._
 
   val spec = suite("Postgres FunctionDef")(
+    test("custom expr rendering") {
+      val query  = select(Overlay("w333333rce", "resou", 3, 5)) from customers
+      val render = renderRead(query) //SELECT overlay(w333333rce placing resou from 3 for 5) FROM customers
+      assert(render)(equalTo("SELECT overlay('w333333rce' placing 'resou' from 3 for 5) FROM customers"))
+    } @@ ignore, //todo fix rendering of strings
```

Comment:
Wouldn't it be enough to just do
`implicit def literal(str: String): Expr[Features.Literal, Any, String] = Expr.Literal(s"'${str}'")`
in expr?

Reply:
this would break once string rendering is fixed - please see https://github.com/zio/zio-sql/pull/307#discussion_r528286267